doc/ref_model/chapters/chapter04.md

```diff
@@ -262,6 +263,32 @@ Table 4-10 shows performance measurement capabilities.
 
 <p align="center"><b>Table 4-11:</b> Cloud Infrastructure management Resource Management Measurements</p>
 
+<a name="4.1.7"></a>
+### 4.1.7 Acceleration/Offload API Requirements 
+HW Accelerators and Offload functions with abstracted interfaces are preferred and can functionally be interchanged, but their characteristics might vary and it is likely that the CNFs/VNFs and Virtualization Infrastructure will have certification requirements for the different implementations. It is also often needed to have a SW implementation of these functions with the same abstracted interfaces for the deployment situations when there are no more HW Accelerator or Offload functions available.
+ 
+For Accelerators and Offload functions with externally exposed differences in their capabilities or management functionality these differences must be clear through the management API either explicit for the differing functions or implicit through the use of a unique APIs.
+ 
+Regardless of exposed or internal different capabilities and characteristics, the operators generally require a choice of multiple implementations also for Accelerators and Offload function realization, which drive the need for ease of portability in between implementations and vendors.
+ 
+The following table of requirement are derived from the need of the VNF/CNF applications, Cloud Infrastructure and operators need for multiple realizations of HW Acceleration and Offload functions that could also be implemented through a software implementation when no special hardware is available. These should be taken into any Reference Architecture to ensure that the different implementations on the market are as aligned as possible in their interfaces and that HW Acceleration and Offload gets an efficient eco system of accelerators that compete on their technical merits and not through obscured or proprietary interfaces.
+
+**Table 4-12** shows Acceleration/Offload API Capabilities.
+
+<a name="Table4-12"></a>
+
+| Ref       | Acceleration/Offload API Capability | Unit   | Definition/Notes                                                                            |
+|-----------|--------------------------------------------|--------|---------------------------------------------------------------------------------------------|
+| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardized interfaces to the Acceleration/Offload function that enables multiple HW and SW implementation of the accelerated/offloaded function from multiple vendors in the Cloud Infrastructure                        |
```

Comment:
Do you have any Accelerator standard i/f in mind in Lakelse timeframe?

Reply:
RM requirements intent and role is mainly to give the RA:s input of their more detailed requirements and specifications. We (RM) would not be in a position to drive the Lakelse standards adoption and realization roadmaps. Having said that, examples of existing protocols would be virtio, NVMe (emulating a physical drive in a SmartNIC/DPU/IPU), vSwitch offload (generally OVS) and some crypto both for in-line and look-aside encryption.
Then there are areas where a wanted standardization do not really have an existing standard on any level e.g. SR-IOV that neither have a higher layer accepted/adopted standard interface, e.g. how to support high performance secondary networking to the CNF applications. required by many Telco applications.

Reply:
This would practically mean the introduction a set of gaps to the RA-s. If this is okay for RM I must accept it.

Reply:
Yes, that is correct.
This is inline with what we do in RM as well, where we try to follow the evolution (here of HW Accelerators) and apply the Anuket basic principles of decoupling of 1. Cloud Infrastructure implementations and VNF/CNF application implementations through rules, guidelines and interfaces as well as 2. HW Layer and Virtualization Layer inside the Cloud Infrastructure.
If these requirements on RM level are accepted for Lakelse, then they will be input for respective RA in next release to judge what are mature and implementable enough to be in that next release. 
If they can be described as Gaps already in the Lakelse RA-1 and RA-2 documents that would naturally be good, but probably not necessary.